src/core/connection.ts

```diff
@@ -1,4 +1,89 @@
-import { ConnectionStatus } from './connection-status.js';
+import * as Ably from 'ably';
+
+import { Logger } from './logger.js';
+import EventEmitter from './utils/event-emitter.js';
+
+/**
+ * Default timeout for transient states before we attempt handle them as a state change.
```

Comment:
CodeRabbit's `to` is still missing haha. It didn't catch you after you said it's done.